Signal/src/ViewControllers/ConversationView/ConversationInputToolbar.m

```diff
@@ -329,106 +289,38 @@ - (BOOL)isInputTextViewFirstResponder
     return self.inputTextView.isFirstResponder;
 }
 
-- (void)ensureContentConstraints
+- (void)ensureShouldShowVoiceMemoButtonAnimated:(BOOL)isAnimated
 {
-    [NSLayoutConstraint deactivateConstraints:self.contentContraints];
```

Comment:
With a stack view, this "rebuild the world" method is mostly unnecessary. As previously written, it wasn't reasonable to animated between the "send" and "microphone" button.